src/validators.rs

```diff
@@ -57,3 +86,12 @@ pub async fn get(client: &Client, address: &str) -> Result<Validator> {
 pub async fn stats(client: &Client) -> Result<Stats> {
     client.fetch("/validators/stats", NO_QUERY).await
 }
+
+/// Get rewards for a validator
+///
+/// Returns rewards for a given validator per reward block the validator is in,
```

Comment:
Definitely a nit but you're being inconsistent with your comment widths. Lined 92 gets newlined at about 80 but line 93 goes all the way to 109?